server/src/main/java/io/spine/server/integration/IntegrationBus.java

```diff
@@ -238,22 +241,22 @@ protected void store(Iterable<ExternalMessage> messages) {
     public void register(ExternalMessageDispatcher<?> dispatcher) {
         super.register(dispatcher);
 
-        // Remember the message types, that we have been subscribed before.
-        final Set<ExternalMessageClass> requestedBefore = subscriberHub.keys();
+        // Remember the channels, that we have been subscribed before.
+        final Set<ChannelId> requestedBefore = subscriberHub.keys();
 
-        // Subscribe to incoming messages of requested types.
+        // Subscribe to incoming messages from the requested channels.
```

Comment:
The problem with the fixes in the comments is that `IntegrationBus` subscribes to messages. And sends requests to other parties (such as remote `IntegrationBus` instances) to tell them that the messages of certain types are needed.

The changes made to the comments mostly destroy this meaning. Now, for some reason, `IntegrationBus` subscribes to some channels, and that's it. Totally unclear to me.